packages/core/useStorage/index.ts

```diff
@@ -218,6 +215,14 @@ export function useStorage<T extends(string | number | boolean | object | null)>
   }
 
   function update(event?: StorageEvent) {
+    if (event && event.storageArea !== storage)
+      return
+
+    if (event && event.key === null) {
+      data.value = rawInit
+      return
+    }
```

Comment:
According to documentation, when key is null, it means that storage has been cleared